docs/core/tools/global-tools-how-to-create.md

```diff
@@ -1,188 +1,189 @@
 ---
-title: How to create a .NET Core Global Tool
-description: Describes how to create a Global Tool. The Global Tool is a console application that is installed through the .NET Core CLI.
-author: Thraka
-ms.author: adegeo
-ms.date: 08/22/2018
+title: How to create a .NET Core tool
+description: Learn how to create a .NET Core tool. A tool is a console application that is installed by using the .NET Core CLI.
+author: tdykstra
+ms.author: tdykstra
+ms.date: 02/12/2020
 ---
 
-# Create a .NET Core Global Tool using the .NET Core CLI
+# Create a .NET Core tool using the .NET Core CLI
 
-This article teaches you how to create and package a .NET Core Global Tool. The .NET Core CLI allows you to create a console application as a Global Tool, which others can easily install and run. .NET Core Global Tools are NuGet packages that are installed from the .NET Core CLI. For more information about Global Tools, see [.NET Core Global Tools overview](global-tools.md).
+**This article applies to:** ✔️ .NET Core 3.x SDK
 
-[!INCLUDE [topic-appliesto-net-core-21plus.md](../../../includes/topic-appliesto-net-core-21plus.md)]
+This tutorial teaches you how to create and package a .NET Core tool. The .NET Core CLI lets you create a console application as a tool, which others can install and run. .NET Core tools are NuGet packages that are installed from the .NET Core CLI. For more information about tools, see [.NET Core tools overview](global-tools.md).
 
-## Create a project
-
-This article uses the .NET Core CLI to create and manage a project.
-
-Our example tool will be a console application that generates an ASCII bot and prints a message. First, create a new .NET Core Console Application.
+The tool that you'll create is a console application that takes a message as input and displays the message along with lines of text that create the image of a robot.
 
-```dotnetcli
-dotnet new console -o botsay
-```
+This is the first in a series of three tutorials. In this tutorial, you create and package a tool. In the next two tutorials you [use the tool as a global tool](global-tools-how-to-use.md) and [use the tool as a local tool](local-tools-how-to-use.md).
 
-Navigate to the `botsay` directory created by the previous command.
+## Prerequisites
 
-## Add the code
+- [.NET Core SDK 3.1](https://dotnet.microsoft.com/download) or a later versions.
+- A text editor or code editor of your choice.
 
-Open the `Program.cs` file with your favorite text editor, such as `vim` or [Visual Studio Code](https://code.visualstudio.com/).
+## Create a project
 
-Add the following `using` directive to the top of the file, this helps shorten the code to display the version information of the application.
+1. Open a command prompt and create a folder named *botsay*.
 
-```csharp
-using System.Reflection;
-```
+1. Navigate to the folder you created and type the following:
```

Comment:
```suggestion
1. Navigate to the folder you created and enter the following command:
```

More accessibility friendly to say enter instead of type.